kdbg/xsldbgdriver.cpp

```diff
@@ -58,6 +58,9 @@ static XsldbgCmdInfo cmds[] = {
     {DCexamine, "print 'x %s %s'\n", XsldbgCmdInfo::argString2},
     {DCinfoline, "print 'templates %s:%d'\n", XsldbgCmdInfo::argStringNum},
     {DCdisassemble, "print 'disassemble %s %s'\n", XsldbgCmdInfo::argString2},
+#ifdef Q_PROCESSOR_X86
+    {DCsetdisassflavor, "set disassembly-flavor %s\n", XsldbgCmdInfo::argString},
+#endif
```

Comment:
Have a look at how `DCassemble` is treated. Do the same with `DCsetdisassflavor`.

Reply:
Do you mean `DCdisassemble`? Is the problem about  `XsldbgCmdInfo::argString`, I don't understand. Also, should I do the same for GdbDriver, too?

Reply:
Do you mean something like this?
```cpp
    {DCsetdisassflavor, "print 'set disassembly-flavor %s %s'\n", XsldbgCmdInfo::argString2},
```

Reply:
Yes, but only the same number of `%s` for arguments that you also need for the Gdb variant. That would be one, not two.

Reply:
Sure, yes. May I ask what is the purpose of this code? I am afraid I don't quite understand it.

Reply:
A decade or two ago someone extended KDbg to be able to debug XSL using xsldbg. This code is slated for removal. But since it's still there, we should not break it knowingly.

Reply:
That makes sense. I implemented an infrastructure to get (and set) debuggers target architecture. I am handling it only for `GdbDriver`. Should I handle it for `XsldbgDriver` too?